### PR TITLE
MPA modification for HPO

### PR DESCRIPTION
**Pull request by eunwoosh**
This PR implemented as bellow

- save current_iters in after_epoch() and after_iter() of the OTEProgressHook class
- pass the curren_iters to time_monitor, then it's used by HPO.

### CODE REVIEW

File: mpa/modules/hooks/__init__.py [hunk 1]
```diff
@@ -18,3 +18,4 @@
 from . import early_stopping_hook
 from . import progress_update_hook
 from . import logger_replace_hook
+from . import save_initial_weight_hook
```
Comment (harimkang) [inline mpa/modules/hooks/__init__.py]: There seems to be a flake8 issue.

File: mpa/cls/trainer.py [hunk 2]
```diff
@@ -113,18 +113,22 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
         # cfg.dump(osp.join(cfg.work_dir, 'config.yaml')) # FIXME bug to save
         # logger.info(f'Config:\n{cfg.pretty_text}')
 
+        # additional callback
+        save_model_callback = kwargs.get('save_model_callback')
```
Comment (harimkang) [inline mpa/cls/trainer.py]: This is a callback for HPO with OTE and it seems not to be used in MPA, is it correct?

**Review by harimkang — CHANGES_REQUESTED** [review]
I will approve it after solving flake8 issue :)

File: mpa/cls/trainer.py (hunk 2 above)
Comment (eunwoosh) [reply mpa/cls/trainer.py]: yes, you're right. I removed all of save_model_callback related things.

File: mpa/modules/hooks/__init__.py (hunk 1 above)
Comment (eunwoosh) [reply mpa/modules/hooks/__init__.py]: You mean f-srting issue in save_initial_weight_hook, right?

**Review by harimkang — APPROVED** [review]
LGTM

**Pull request merged**